CleverRaven/Cataclysm-DDA
# added roof to A/C carrier

#### Summary
None



#### Purpose of change
Provide the A/C carrier with an explicit roof in preparation for the removal of roof magic.



#### Describe the solution
Add another level to the carrier containing the roof (and some decorations on it).
Also fixed a broken railing.



#### Describe alternatives you've considered
Make the roof out of the same stuff as the deck is made out of.



#### Testing
- Hack add_roofs to disable the magic.
- Debug spawn a carrier special.
- Teleport to it.
- Teleport around to inspect it.
- Verify there is no roof on the tower.
- Add the roof and repeat the above (less teleporting around for inspection).
- Declare the roof to be good.
- Restore the hacked code.

![Screenshot (379)](https://github.com/CleverRaven/Cataclysm-DDA/assets/22739822/c7977a93-a542-43ac-bf34-219fff868150)



#### Additional context
It's a bit silly to have zombies spawn in the air in droves. I don't think The Weather Girls would approve of this twist on their hit.
